docs/Developers/Newcomers_Guide.rst

```diff
@@ -449,7 +445,7 @@ Now you should `tell us about your experience <https://coala.io/newform>`_ and
 go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
 
-.. note::
+.. caution::
```

Comment:
Maybe we want a new tag "critical" or so, because this is so damn important imo 😅

Reply:
I will add critical ;) Is that okay ?

Reply:
@Makman2 I am using tag "critical" for this one. Can you suggest me the background color to be used with this? Red is already used for important and yellow for caution.